apps/prairielearn/src/pages/administratorSettings/administratorSettings.html.ts

```diff
@@ -131,9 +131,45 @@ export function AdministratorSettings({ resLocals }) {
                 <button type="button" class="btn btn-outline-info">Info</button>
                 <button type="button" class="btn btn-outline-dark">Dark</button>
               </div>
-              <div class="p-4 bg-dark">
+              <div class="mb-4 p-4 bg-dark">
                 <button type="button" class="btn btn-outline-light">Light</button>
               </div>
+              <h3>Custom colors</h3>
```

Comment:
Here's how this looks when rendered:

<img width="1254" alt="Screenshot 2024-10-28 at 13 31 02" src="https://github.com/user-attachments/assets/5cb7663d-4c86-4621-92c2-b55993d04539">